<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <link rel="stylesheet" href="style/base.css" type="text/css" />
    </head>
    <body style="margin: 0">
        <div class="container" style="text-align:justify">
            <h2>ProteoRE : release notes</h2>
     		<h3><font color="0000FF">Release 25th June 2021</font></h3>
            	<ul>
                <li>ProteoRE server: CPU and memory resources have been extended and jobs management configuration revisited to better satisfy user needs</li>
		     
				<li>Update of resources used by ProteoRE tools (<a target="_blank" href="https://proteore.org/static/data_source.html">more details here</a>):</li>
					<ul>
					<li>GO terms source files for human, mouse and rat ("Compute functional profiles", "Classification and enrichment analysis", "Biological theme comparison", "Enrichment analysis")</li>  
					<li>ID mapping files have been updated ("ID-converter")</li>
					<li>neXtprot annotations were updated ("Add protein features [neXtProt, Human]")</li>
					<li>Expression data sources from Human Protein Atlas ("Get expression profiles", "Add expression data", "Build tissue-specific expression dataset")</li>
					<li>PPI data from BioGRID ("Build Protein interaction network")</li>
					</ul>
                <li>KEGG pathways mapping and visualization [PathView] tool: fixed bug associated with KEGG map visualization</li></b>
				<li><b>New tool</b>: <a target="_blank" href="https://proteore.org/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fproteore%2Fproteore_go_terms_enrich_comparison%2Fgo_terms_enrich_comparison%2F2020.02.04">Biological themes comparison (Human, Mouse, Rat)[clusterProfiler]</a>: try it out!</li>
				<li>ProteoRE workflows have been updated accordingly:</li>
					<ul>
					<li>Tutorial 1: <a target="_blank" href="https://proteore.org/u/proteore/w/proteoreproteomeannotationtutorial-release-20">Annotating a protein list identified by LC-MS/MS experiments (release 2.0)</a> (from PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/29189203/">29189203</a>)</li>
					<li>Tutorial 2: <a target="_blank" href="https://proteore.org/u/proteore/w/workflow-constructed-from-history-proteorebiomarkerscandidatetutorial">Biomarkers candidate identification</a> (from PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/30852829/">30852829</a>)</li>
					<li><a target="_blank" href="https://proteore.org/u/proteore/w/imported-workflow-pancreaticcancerbiomarkersselectionstrategyshared">Workflow for the Selection of Pancreatic Cancer Biomarkers</a> (from PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/31538697/">31538697</a>)</li>
					<li><a target="_blank" href="https://proteore.org/u/yvdb/w/workflow-constructed-from-history-mimbgoprotocolproteore2021">Annotation and GO Enrichment Analysis of a List of Differentially Expressed Proteins</a> (from PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/34236662/"</a>34236662</a>)</li>
					</ul>
              	</ul>
		<h3><font color="0000FF">Release 5th February 2020 (version 2.1)</font></h3>
            <b>Update/Enhancement:</b>
                <ul>
                <li>Tools panel was reorganized (now four sections) and some tools were renamed to make them more explicit (thanks to ProteoRE’s users feedback)</li>
		<li>Genome wide annotation source files for human, mouse and rat have been updated for all functional analysis (GO) tools</li>  
		<li>All ID mapping files have been updated (18/12/2019)</li>
		<li>Expression data source files from HPA (and related tools) have been fully updated</li>
		<li><b>Build tissue-specific expression dataset [Human Protein Atlas]</b> tool: the interface has been modified to account for additional tissues form the the latest release of HPA, read the UserDoc section of this tool</li>	
		<li><b>ID converter</b> tool: gene name symbol is now included and mapping files were updated accordingly for <i>Homo sapiens, Mus musculus, Rattus norvegicus</i> (UniProt release 18/12/2019).</li>
                <li><b>Venn diagram [JVenn]</b> tool: : bug due to one extra-line counting is now fixed</li>
                <li><b>KEGG pathways mapping and visualization [PathView]</b> tool: fixed bug associated with KEGG map visualization</li></b>
		</ul>
             <b>New tools:</b>
		<ul>
                <li><b><a target="_blank" href="http://www.proteore.org/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fproteore%2Fproteore_prot_features_3orga%2Fprot_features_3orga%2F2020.01.23">Add protein features [UniProt, Human, Mouse, Rat]</a></b>: This tool retrieves protein features (e.g. subcell location, topological domain, pathway, disease) from the UniProt database (Reference Proteome) to annotate your protein IDs list</li>
                <li><b><a target="_blank" href="http://www.proteore.org/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvolcanoplot%2Fvolcanoplot%2F0.0.3">Volcano plot</a></b>: this tool allows to draw a Volcano Plot and to add labels. Very useful for differential analysis to quickly visualize genes/proteins and to identify the most meaningful changes between two conditions</li>
                <li><b><a target="_blank" href="http://migale.jouy.inra.fr/proteore/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fproteore%2Fproteore_go_terms_enrich_comparison%2Fgo_terms_enrich_comparison%2F2020.02.04">Biological themes comparison (Human, Mouse, Rat)[clusterProfiler]</a></b>: this tool performs biological theme comparison by computing enriched functional categories between different protein/gene lists (e.g. in different conditions)</li>
		</ul>
         
	    <h3><font color="0000FF">Release 28th June 2019 (version 2.0)</font></h3>
	    <b>Update/Enhancement:</b>
                <ul>
                <li><b>Filter by keywords and/or numerical value</b>: addition of new option "keep/discard"</li>
                <li><b>ID converter</b>: the <i>Rattus norvegicus</i> species is now available for id conversion; addition of mapping database identifiers files for <i>Homo sapiens</i>, <i>Mus musculus</i>, <i>Rattus norvegicus</i> (UniProt release 05/06/2019). </li>
                <li><b>Add protein features [neXtProt]</b>: addition of release 2019-01-11</li></b>
		<li><b>Statistical analysis of functional profiles (Human, Mouse) [goProfiles]</b>: "Ontology level" option extended up to level 6. </li>
		<li><b>Query pathway database [Reactome]</b>: "species" option extended to <i>Mus musculus</i> and <i>Rattus norvegicus</i></li>
                </ul>
             <b>New tools:</b>
		<ul>
                <li><b><a target="_blank" href="http://www.proteore.org/?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fproteore%2Fproteore_build_protein_interaction_maps%2Fbuild_protein_interaction_maps%2F2019.03.12.2&version=2019.03.12.2&__identifer=22uwwokghi5i">Build Protein-Protein interaction network [BioGRID, BioPlex, HuMAP]</a></b>: This tool allows building interaction maps by mapping your list of protein/gene identifiers on different public resources; the following PPIs resources are available: BioGRID(Homo sapiens, Mus musculus and Rattus norvegicus species); Human Bioplex (V.2.0) and HuMAP (Human Protein Complex Map). The two result files generated (network and nodes attributes) can be used for viewing and further exploration of the resulting protein interactions network by directly importing those in dedicated software (e.g. Cytoscape).</li>
                <li><b>Add protein features (Mouse) [UniProt]</a></b>: now deprecated and replaced by <b>Add protein features [UniProt, Human, Mouse, Rat]</b> tool in the next release (see above).</li>
                </ul>
	   
            <h3><font color="0000FF">Release 18th December 2018 (version 1.2)</font></h3>

            ProteoRE now comprises 15 tools for:
            <ul>
                <li>Data manipulation (e.g. filtering, list comparison, ID conversion)</li>
                <li>Human annotation (protein features, MS/MS observations, expression and tissue-specificity)</li>
                <li>GO terms analysis (functional profiles, enrichment analysis)</li>
                <li>Pathways analysis (Reactome and KEGG pathways exploration)</li>
                <li>Graphical representation (Venn diagram, heat map, bar- and dot-plot…)  </li>
            </ul>

            <h3><font color="0000FF">Release 19th September 2018 (version 1.1)</font></h3>

	    <b>Updates</b>: 
            <ul>
                <li>Filter by keywords or numerical value: You can now filter by a range of values, choose an operator to use when multiple filters are set (AND or OR) and sort your result files</li>
                <li>ID converter: the Mus musculus species is now available for id conversion, update of reference mapping file for id conversion, from Uniprot and neXtProt files (12/09/18)</li>
                <li>goProfiles: Mus musculus is now available as reference species, packages updated, IDs format is now checked. </li>
            </ul>

		<b>New tools</b>:
            <ul>
                <li>Number of MS/MS observations in a tissue : this tool reports the number of times peptides matching a protein have been observed in a given human biological sample using LC-MS/MS proteomics approach. Data were queried from Peptide Atlas release (Jan 2018). </li>
                <li>KEGG pathways coverage : this tool reports pathways concern by the proteins entered. Pathways are found with the Keggrest R library, results are sorted by the number of mapping ids on the number of pathway genes (%). </li>
            </ul>

            <h3><font color="0000FF">Release 7th May 2018 (version 1.0)</font></h3>

            ProteoRE comprises tools for:
            <ul>
                <li>Standard data manipulation (e.g. filtering, sorting, and ranking)Protein list annotation</li>
                <li>Annotation retrieval from databases</li>
                <li>Functional and pathways analysis (cross-comparison, classification and enrichment analysis)</li>
                <li>Graphical representation      </li>
            </ul>
        </div>
    </body>
</html>
   
