<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-88179-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-88179-2');
</script>

        <link rel="stylesheet" href="style/base.css" type="text/css" />
    </head>
    <body style="margin: 0">
        <div class="jumbotron jumbotron-fluid">
             <div class="container">
               <img src="images/proteore.png/" width="200px">
               <font size="+2"><strong>Welcome to ProteoRE !</strong></font>
            </div>
         </div>


<!--
     <div class="container" style="text-align:justify">
            <div class="jumbotron">
                <table style="width:100%">
                    <tr>
                        <th><img src="images/red_warning.png" width="80px"></th>
                        <th style="text-align:center"><font size="3"><strong>Proteore.org will be unavailable for maintenance purpose from the 5th to the 6th Feb 2020.<br/>
                            Thank you for your understanding. </strong></font></th>
                        <th><img src="images/red_warning.png" width="80px"></th>
                    </tr>
                </table>
      </div>
-->
        <div class="container" style="text-align:justify">
            <p class="lead">
        <h3>What is ProteoRE ?</h3>
                ProteoRE (Proteomics Research Environment) is a Galaxy-based platform for the functional analysis and the interpretation of proteomics and transcriptomics data in biomedical research; ProteoRE allows researchers to apply a large range of bioinformatics tools and tailored workflows on their own data using Galaxy capabilities. ProteoRE comprises <b>21 tools</b> organized into four subsections: i) data manipulation and visualization; ii) get features/annotation; iii ) functional analysis; and iv) pathway analysis along with graphical representation. This version also includes Galaxy generic tools and some tools from the GalaxyP project that should be of great help for (re)processing MS-based proteomics data (e.g. peptide/protein identification). 
		<BR>
        
	<h3>ProteoRE new release (24th June 2024) - <a target="_blank" href="https://proteore.org/static/proteore_previous_releases_notes.html">release notes</a></h3>
	
            <ul>
                <li>ProteoRE server: CPU and memory resources have been extended and jobs management configuration revisited to better satisfy user needs</li>
		     
		<li>Update of resources used by ProteoRE tools (<a target="_blank" href="https://proteore.org/static/data_source.html">more details here</a>):</li>
					<ul>
					<li>ID mapping files have been updated ("ID-converter")</li>
					<li>Expression data sources from Human Protein Atlas ("Get expression profiles", "Add expression data", "Build tissue-specific expression dataset")</li>
					<li>PPI data from BioGRID ("Build Protein interaction network")</li>
					</ul>
                			<li>ProteoRE tutorials and workflows updated:</li>
					<ul>
					<li>Tutorial 1: <a target="_blank" href="https://proteore.org/u/proteore/w/proteoreproteomeannotationtutorial-release-20">Annotating a protein list identified by LC-MS/MS experiments (release 2.0)</a> (from PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/29189203/">29189203</a>)</li>
					<li>Tutorial 2: <a target="_blank" href="https://proteore.org/u/proteore/w/workflow-constructed-from-history-proteorebiomarkerscandidatetutorial">Biomarkers candidate identification</a> (from PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/30852829/">30852829</a>)</li>
					<li><a target="_blank" href="https://proteore.org/u/proteore/w/imported-workflow-pancreaticcancerbiomarkersselectionstrategyshared">Workflow for the Selection of Pancreatic Cancer Biomarkers</a> (from PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/31538697/">31538697</a>)</li>
					<li><a target="_blank" href="https://proteore.org/u/yvdb/w/workflow-constructed-from-history-mimbgoprotocolproteore2021">Annotation and GO Enrichment Analysis of a List of Differentially Expressed Proteins</a> (from PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/34236662/"</a>34236662</a>)</li>
					</ul>
		           	</ul>
		 
		<h3>How to access ProteoRE ?</h3>
                This public site is meant for academic usage;  for commercial usage, <a href="mailto:contact@proteore.org">contact us</a>). ProteoRE can currently be used as:  
		        <li><b>Unregistered user</b>: you can analyze your data (e.g. proteomics identification results in tab format) using the available tools. Allocated storage space is limited to 20 Mb and data/history/analyses are not conserved between sessions.</li>
                <li><b>Registered user</b> (password-protected access): we recommend to create a user account. Up to 20 Gb storage space is offered to registered users, along with access to shared resources (data, histories, workflows..). To create an account: click on "Account registration or login" on the main menu bar (upper part of the ProteoRE central panel), click on "registration" and fill out the form. Once the form is submitted, your account is created.</li>

		<h3>ProteoRE Tutorials available in Galaxy Training Network:</h3>
        	<ul>
                <li><a href="https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/proteome_annotation/tutorial.html" target="_blank"><b>Annotating a protein list identified by LC-MS/MS experiment</b></a></li>
                <li><a href="https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/biomarker_selection/tutorial.html" target="_blank"><b>Strategy to select biomarker candidates</b></a></li>
		</ul>
	    
		<h3>How to cite us? </h3>
		If you use resources from ProteoRE, please cite: Mehta S, et al. A Galaxy of informatics resources for MS-based proteomics. Expert Rev Proteomics. 2023, 20(11):251-266. doi: 10.1080/14789450.2023.2265062. PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/37787106/">37787106</a>.  
		<h3>How to contribute? </h3>
		<ul>
		<li><b>As a developper/bioinformatician</b>: ProteoRE team welcomes any contribution via GitHub, please <a href="https://github.com/vloux/ProteoRE" target="_blank">read here</a></li>
                <li><b>As a biologist/clinician</b>: ProteoRE is constantly evolving and we greatly appreciate users feedbacks helping at improving ProteoRE capabilities: <a href="mailto:contact@proteore.org">contact us ! </a></li>
        </ul>
		<h4><a href="https://proteore.org/static/communication.html" target="_blank">Publication, Event, Community links</a></h4>
 
		<h4>Data sources: <a href="https://proteore.org/static/data_source.html" target="_blank">public resources used by ProteoRE's tools</a></h4> 
	
		<h4>Support and contact: <a href="mailto:contact@proteore.org">contact@proteore.org</a></h4>	
	
	</p>
            <footer>
                <h3>Fundings</h3>
                ProteoRE project is a joint national effort between the <a target="_blank" class="reference" href="http://www.france-bioinformatique.fr/en">French bioinformatics Institute (IFB)</a> and the <a target="_blank" class="reference" href="http://www.profiproteomics.fr/">Proteomics French Infrastructure (ProFI)</a> and is partly funded by the <a target="_blank" class="reference" href="http://www.agence-nationale-recherche.fr/en/">French Research Agency</a> (ANR-11-INBS-0013).<br/>
                <BR>
                <img src="images/ifb.png/" width="100px">
                <img src="images/profi.png/" width="100px">
                <img src="images/anr.png/" width="100px"><br/>
                <BR>
		The <a target="_blank" class="reference" href="http://galaxyproject.org/">Galaxy Project</a>
                is supported in part by <a target="_blank" class="reference" href="http://www.genome.gov">NHGRI</a>,
                <a target="_blank" class="reference" href="http://www.nsf.gov">NSF</a>,
                <a target="_blank" class="reference" href="http://www.huck.psu.edu">The Huck Institutes of the Life Sciences</a>,
                <a target="_blank" class="reference" href="http://www.ics.psu.edu">The Institute for CyberScience at Penn State</a>,
                and <a target="_blank" class="reference" href="http://www.jhu.edu/">Johns Hopkins University</a>. We are also very grateful to the Galaxy-P team (University of Minnesota, US) and Bjorn Gr√ºning (University of Freiburg, DE) for their help and constant support. <br/>
                <h3>People</h3>
                Valentin Loux, Yves Vandenbrouck. Alumni: we are very grateful to Virginie Brun, Christophe Caron, David Christiany, Florence Combes, Benoit Gilquin, Lien Nguyen, Lisa Perus, Maud Lacombe, Marianne Tardif, for their contribution to ProteoRE and their enthousiasm.
		        <BR>
            </footer>
        </div>
    </body>
</html>
